components/hmpo-template.njk

```diff
@@ -24,8 +24,13 @@
 {% block header %}
     {% block cookieBanner %}
         {% from "hmpo-cookie-banner/macro.njk" import hmpoCookieBanner %}
-        {{ hmpoCookieBanner({
-            html: translate("govuk.cookieBanner")
+        {{ hmpoCookieBanner(ctx, {
+            title: translate("govuk.cookieBanner.title"),
+            html: translate("govuk.cookieBanner.content") ,
+            accept: translate("govuk.cookieBanner.accept"),
+            preferences: translate("govuk.cookieBanner.preferences"),
+            acceptHref: hmpoAcceptCookies,
+            preferencesHref: hmpoCookiePreferences
```

Comment:
It's tempting to put this component in an `{% if hmpoAceptCookies and hmpoCookiePreferences %}` so its not rendered by default on pages that don't specify the urls.

(the components are used on other sites, most of them will kill the cookieBanner block if they don't want it)

(perhaps show the old component if the urls aren't specified?)

Reply:
Alternatively, perhaps the component can be explicitly used by shared lib, and the cookies etc aren't set here, but by middleware, and it shown by shared lib?

It made sense for it here when it was really generic message, but now its a bit more custom to DCS

Reply:
That is a good suggestion - I have moved the banner to shared lib instead.